extension.neon

```diff
@@ -1,3 +1,14 @@
+parameters:
+	magento:
+		checkCollectionViaFactory: true
+		checkServiceContracts: true
```

Comment:
@shochdoerfer Instead of adding options to rules themselves to turn them off by not reporting anything, instead you can use the `conditionalTags` section like here: https://github.com/Roave/no-floaters/blob/master/rules.neon#L10-L12

Reply:
Ah nice, even better. Thanks!